data/campaigns/Winds_of_Fate/scenarios/05_The_Three_Sisters.cfg

```diff
@@ -0,0 +1,840 @@
+        [fire_event]
+            name=saurians rescued
+        [/fire_event]
+        [fire_event]
+            name=loyalist dialog
+        [/fire_event]
+    [/event]
+
+    [event]
+        name=saurians rescued
+        # dialog with saurians
+        # wmllint: who ARINEXIS is Arinexis
+        [unit]
+            {ARINEXIS}
+            side=5
+            x,y=8,30
+            facing=nw
+        [/unit]
+        # wmllint: who ZEDRIX is Zedrix
+        [unit]
+            {ZEDRIX}
+            side=5
+            x,y=9,31
+            facing=nw
+        [/unit]
+        [message]
+            speaker=Zedrix
+            image_pos=right
+            mirror=yes
+            message= _ "He returnsss! Noble Shek’kahan returnss from ssspirit realm to ssave usss! (<i>chirping sob</i>)" # no spellcheck
+        [/message]
+        [message]
+            speaker=Gorlack
+            message= _ "Reshan.
+
+These... are <b>not</b> hatchlings."
+        [/message]
+        [message]
+            speaker=Reshan
+            message= _ "By the Winds, they are not.
+Indeed, their resemblance is so strikingly of hatchlings."
+        [/message]
+        [message]
+            speaker=Arinexis
+            image_pos=right
+            mirror=yes
+            message= _ "Ssso thesse are Fire Dragonsss, eh Zedrix?
+
+Hail, dragonss!  (<i>rapid clicking</i>)  Down here!" # no spellcheck
+        [/message]
+        [message]
+            speaker=Reshan
+            message= _ "They are receiving us."
+        [/message]
+        [message]
+            speaker=Arinexis
+            image_pos=right
+            mirror=yes
+            message= _ "Hail-loooo!  Dragonss, can you ssspeak?  Zedrix, you ssaid dragons can ssspeak!" # no spellcheck
+        [/message]
+        [message]
+            speaker=Gorlack
+            message= _ "Little ones!
+
+You will do as I command!"
+        [/message]
+        [message]
+            speaker=Zedrix
+            image_pos=right
+            mirror=yes
+            message= _ "By every sssign upon firmament, we hear you Shek’kahan!  You are our moonlight!" # no spellcheck
+        [/message]
+        [modify_unit]
+            [filter]
+                race=lizard
+            [/filter]
+            side=1
+        [/modify_unit]
+    [/event]
+
+    [event]
+        name=loyalist dialog
+        [delay]
+            time=1000
+        [/delay]
+        [message]
+            speaker=Elyn
+            image_pos=right
+            mirror=yes
+            message= _ "Rhynnin has fallen, just as I feared he would. So eager to serve his King and so very proud of his quest here; he underestimated the treachery of this accursed place and our strange enemy that would choose to shelter here.
+
+Men, I am now in command of this expedition. No more lone heroics. Signal the eastern detachment that we will try and reunite with them along the isle's southern shore."
```

Comment:
'try and reunite' -> 'try **to** reunite'

Reply:
change applied as suggested